packages/app/src/components/collapsible/collapsible-button.tsx

```diff
@@ -55,62 +92,116 @@ export const CollapsibleButton = ({
   );
 };
 
-const Container = styled(Box).attrs({ as: 'section' })(
+const ButtonContainer = styled.div(
+  css({
+    display: 'flex',
+    justifyContent: 'center',
+    mx: 'auto',
+  })
+);
+
+const Container = styled(Box).attrs({ as: 'section' })<{
+  isOpen: boolean;
+  buttonWidth: number;
+  contentWidth: number;
+  contentHeight: number;
+  buttonHeight: number;
+  clipPathCalculation: string;
+}>((x) =>
   css({
-    borderRadius: 1,
     position: 'relative',
-    display: 'inline-flex',
-    flexDirection: 'column',
     padding: 0,
-    margin: '0 auto',
     transitionProperty: 'height',
-    transitionDuration: '0.5s',
-    bg: 'tileGray',
-    width: '100%',
+    transitionDuration: '0.4s',
+    willChange: 'height',
 
-    //button
+    // Button
     '[data-reach-disclosure-button]': {
-      position: 'relative',
-      px: asResponsiveArray({ _: 2, sm: 4 }),
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      overflow: 'hidden',
+      zIndex: 1,
+      width: x.isOpen ? '100%' : 'fit-content',
+      border: '1px solid',
+      borderRadius: 1,
+      borderColor: x.isOpen ? 'rgba(0, 0, 0, 0)' : 'lightGray',
+      px: asResponsiveArray({ _: 1, sm: 3 }),
       py: 3,
-      border: 'none',
       background: 'none',
+      color: 'black',
       font: 'inherit',
-      color: 'blue',
-      cursor: 'pointer',
       fontWeight: 'bold',
-      zIndex: 1,
-      width: '100%',
+      cursor: 'pointer',
+      transition: 'color 0.2s ease-out',
+
+      '&:hover': {
+        color: 'blue',
+
+        svg: {
+          fill: 'blue',
+        },
+
+        '&:before': {
+          borderColor: x.isOpen ? 'lightGray' : 'data.primary',
+        },
+      },
+
+      // Outside border
+      '&:before': {
+        transition: 'width 0.4s',
+        position: 'absolute',
+        top: 0,
+        width: x.isOpen ? '100%' : x.buttonWidth,
+        height: '100%',
+        border: '1px solid',
+        borderRadius: 1,
+        borderColor: 'lightGray',
+        zIndex: -1,
+        content: '""',
+        pointerEvents: 'none',
+      },
     },
+
     '[data-reach-disclosure-button]:focus': {
       outlineWidth: '1px',
       outlineStyle: 'dashed',
-      outlineColor: 'blue',
-    },
-    '[data-reach-disclosure-button][data-state="open"]:after': {
-      content: '""',
-      position: 'absolute',
-      bottom: 0,
-      left: 0,
-      right: 0,
-      bg: 'lightGray',
-      height: '1px',
+      outlineColor: x.isOpen ? 'rgba(0, 0, 0, 0)' : 'blue',
```

Comment:
```suggestion
      outlineColor: x.isOpen ? 'transparent' : 'blue',
```

Or better, use the rgb values for `blue`, and use that as the transparent color?

Reply:
The blue border doesn't animate/transition so in my opinion transparent is a bit nicer and easier to read. 👍